visualisation/nzvm.py

```diff
@@ -130,14 +135,34 @@ def main():
     if length != nx*ny*nz:
         print("Error: Inconsistent VM size. Check model and nx, ny, nz parameters and try again. Expected",nx*ny*nz,"values, got",len(array))
         quit()
+
+    if args.perturbation:
+        pert_length, pert_array = load_data(args.perturbation, nx, ny, nz)
+        if pert_length != nx*ny*nz:
+            print("Error: Inconsistent perturbation file size. Check model and nx, ny, nz parameters and try again. Expected",nx*ny*nz,"values, got",len(pert_array))
```

Comment:
Instead should perhaps say something about the perturbation matching the VM. If it has gotten to this point, the VM matches nx, ny, nz.